modules/gitops-release-notes-1-10-0.adoc

```diff
@@ -0,0 +1,120 @@
+====
+
+* With this update, the {gitops-title} Operator deploys three monitoring dashboards in the Administrator perspective of the web console. The three dashboards are: *GitOps Overview*, *GitOps Components*, and *GitOps gRPC*. To access these dashboards, go to *Observe* → *Monitoring*. link:https:https://issues.redhat.com/browse/GITOPS-1767[GITOPS-1767]
++
+[NOTE]
+====
+Disabling or changing the content of the dashboards is not supported.
+====
+
+* With this update, the {gitops-title} Operator now gathers and exports metrics data that can be handled by Prometheus. The metrics data collected by the server include the following:
++
+** The total number of Argo CD instances managed and reconciled.
+** The number of Argo CD instances in a given state, such as pending, available, among others. 
+** The total time for the reconciliation loops to complete per instance,
+** The amount of resources used such as CPU, memory, and number of goroutines.
++
+To access these metrics, go to the *Observe* tab on the web console, and run queries against the monitoring stack. link:https://issues.redhat.com/browse/GITOPS-2645[GITOPS-2645]
++
+[NOTE]
+====
+You need to install the {gitops-title} Operator in the default `openshift-gitops-operator` namespace, in order to have these metrics automatically available.
+====
+
+* Previously, timestamps were presented in a Unix epoch format. With this update, the timestamps are changed to RFC3339 format, for example: 2023-06-27T07:12:48-04:00, to improve overall readability. link:https://issues.redhat.com/browse/GITOPS-2898[GITOPS-2898]
+
+* With this update, the default Argo CD instance in the `openshift-gitops` namespace has restricted permissions for non-admin users by default. This improves security because non-admin users no longer have access to sensitive information. However, as an administrator, you can set permissions and grant non-admin users access to the resources managed by the default `openshift-gitops` Argo CD instance by configuring your Argo CD RBAC. This change only applies to the default {gitops-title} instance. link:https://issues.redhat.com/browse/GITOPS-3032[GITOPS-3032]
+
+* With this update, the default installation namespace for {gitops-title} Operator has been changed to its own namespace called `openshift-gitops-operators`. You can still choose the old default installation namespace, `openshift-operators`, through a drop-down menu available in the OperatorHub UI at installation time. You can also enable cluster monitoring on the new namespace by selecting the check box, which makes the Operator's performance metrics accessible within the {product-title} web console. link:https://issues.redhat.com/browse/GITOPS-3073[GITOPS-3073]
++
+[NOTE]
+====
+The {gitops-title} Operator's metrics are only available when the Operator is installed in the namespace other than `openshift-operators`.
```

Comment:
this doesn't seem very clear to me... it's only available when the operator is installed in the `openshift-gitops-operator` namespace, not just any other namespace.

Reply:
I have updated content. It now clearly state the default namespace.